nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -225,6 +225,13 @@
      The fourStore and fourStoreEndpoint modules have been removed.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     Polkit no longer has the user of uid 0 (root) as an admin identity.
+     We now follow the upstream default of only having every member of the wheel
+     group admin privileged. Before it was root and members of wheel.
```

Comment:
The only thing I'd add here is the user-observable benefit, like

> The positive outcome of this is that `pkexec` GUI popups or terminal prompts will no longer require the user to choose between two essentially equivalent choices (whether to perform the action as themselves with `wheel` permissions, or as the `root` user).

Reply:
Thanks, will add this.